gateway/tests/fixtures/fixtures.json

```diff
@@ -49,7 +49,7 @@
       "password": "pbkdf2_sha256$390000$kcex1rxhZg6VVJYkx71cBX$e4ns0xDykbO6Dz6j4nZ4uNusqkB9GVpojyegPv5/9KM=",
       "is_active": true,
       "groups": [
-          101
+          100
```

Comment:
This is to change the user group to the runner one that has the `run` permission.